deliveries/zeropool-2-snjax.md

```diff
@@ -0,0 +1,23 @@
+# Evaluation
+
+
+
+- **Status:** Accepted
```

Comment:
```suggestion
```

Reply:
That's reserved for evaluations ;)

Reply:
@snjax, I submitted a PR to your fork to adapt the application to the right template. This issue will be resolved if you merge it.

Reply:
Thank you for the PR.